packages/dx-react-grid/docs/reference/table-header-row.md

```diff
@@ -43,7 +43,7 @@ Field | Type | Description
 column | [Column](#column) | A column object
 allowSorting | boolean | If true, an end-user can change sorting by a column
 sortingDirection? | 'asc' &#124; 'desc' | Specifies the column sort order
-changeSortingDirection | ({ keepOther: boolean, cancel: boolean }) | Changes column sort direction. Keeps the existing sorting if `keepOther` is set to `true`. Cancels sorting by the current column if `cancel` is set to true.
+changeSortingDirection | ({ keepOther: boolean, cancel: boolean, scope: Array&lt;String&gt; }) | Changes column sort direction. Keeps the existing sorting if `keepOther` is set to `true`. Cancels sorting by the current column if `cancel` is set to true. `scope` is an array which contains `columnName` among that sorting is carried out.
```

Comment:
According to the code, scope parameter is not possible to pass to that function.